docs/admin/release_notes/version_3.2.md

```diff
@@ -9,6 +9,12 @@ Device Lifecycle Management App version 3.2.0 adds support for ArubaOS and PanOS
 
 ## [v3.2.0 (2025-10-24)](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/releases/tag/v3.2.0)
 
+!!! warning
```

Comment:
Should we move these up to the release overview section? I think it's probably fine either way, just wondering if it would help for people who only read the release overview of these files.